app.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'lib/item_repository'
+require_relative 'lib/order_repository'
+require_relative 'lib/database_connection'
+
+class Application
+
+  def initialize(io, item_repository, order_repository)
+    DatabaseConnection.connect('shop_test')
+    @io = io
+    @item_repository = item_repository
+    @order_repository = order_repository
+  end
+
+  def run
+    @io.puts("Welcome to the shop managment program!")
+    @io.puts("")
+    @io.puts("1 = list all shop items")
+    @io.puts("2 = create a new item")
+    @io.puts("3 = list all orders")
+    @io.puts("4 = create a new order")
+
+    user_choice(@io.gets.chomp)
+
+    return nil
+  end
+
+  def user_choice(input)
+    case input
+    when '1'
+      all_items
+    # when '2'
+    #   create_item
+    when '3'
+      all_orders
+    # when '4'
+    #   create_order
```

Comment:
Maybe be a bit clearer on any outstanding methods or work that is yet to be completed.